conf/routes

```diff
@@ -19,7 +19,10 @@ GET /api/campaigns/bootstrap    controllers.CampaignApi.bootstrapData
 GET /api/campaigns              controllers.CampaignApi.getAllCampaigns
 GET /api/campaigns/:id          controllers.CampaignApi.getCampaign(id: String)
 PUT /api/campaigns/:id          controllers.CampaignApi.updateCampaign(id: String)
-GET /api/campaigns/:id/analytics          controllers.CampaignApi.getCampaignAnalytics(id: String)
+GET /api/campaigns/:id/analytics       controllers.CampaignApi.getCampaignAnalytics(id: String)
+GET /api/campaigns/:id/notes    controllers.CampaignApi.getCampaignNotes(id: String)
+POST /api/campaigns/:id/notes   controllers.CampaignApi.addCampaignNote(id: String)
+PUT /api/campaigns/:id/notes    controllers.CampaignApi.updateCampaignNote(id: String)
```

Comment:
I'm not sure what you mean, I've now changed the put route as you suggested and left the get route as it is since it is getting all the notes in the campaign.